### DISCUSSION TITLE
How can i show the file tree automatically whenever i start nvim ?

### DISCUSSION THREAD

**Discussion by noamso1** [NvChad/NvChad#2418] [General]
i've tried add in my custom init.lua this line:
`vim.cmd("autocmd VimEnter * NvimTreeToggle")`
and it sort of worked but created a lot of bugs and errors..

**Comment by noamso1** [comment]
Closing - Duplicate... I found a similar question in an earlier discussion.

**Comment by siduck** [comment]
```bash
nv() {
  [ -n "$1" ] && [ -d "$1" ] && cd "$1" && shift
  nvim +'lua require("nvim-tree.api").tree.open()'  "$@"
}
```
shell config
